File: examples/js/delete__.js

const authToken = process.env.AIOPS_TOKEN || 'YOUR_TOKEN';
// 示例：DELETE /
fetch(`http://localhost:8000/api/v1/`, {
    method: 'DELETE',
    headers: { Authorization: `Bearer ${authToken}` }
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));

File: examples/python/delete__.py
# -*- coding: utf-8 -*-
import requests

# 示例：DELETE /
# 使用 DELETE 方法请求 /
url = "http://localhost:8080/api/v1/"
try:
    response = requests.delete(url)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)
